cluster/src/main/scala/io/snappydata/metrics/SnappyMemberMetrics.scala

```diff
@@ -21,19 +21,19 @@ package io.snappydata.metrics
 
 import io.snappydata.metrics.SnappyMetricsSystem.{createGauge, updateHistogram}
 import com.pivotal.gemfirexd.internal.engine.ui.MemberStatistics
-import java.util
-
+import com.pivotal.gemfirexd.internal.engine.Misc
 import io.snappydata.Constant
 
 object SnappyMemberMetrics {
 
-  def convertStatsToMetrics(member: String, memberDetails: MemberStatistics,
-       allMetaEntries: util.Map[String, String]) {
+  def convertStatsToMetrics(member: String, memberDetails: MemberStatistics) {
+
     val shortDirName = memberDetails.getUserDir.substring(
       memberDetails.getUserDir.lastIndexOf(System.getProperty("file.separator")) + 1)
 
-    val memberUuid = allMetaEntries.
-        get(Constant.MEMBER_ID_PREFIX + shortDirName + member + "__").toString
+    val memberUuid = Misc.getMemStore.getMetadataCmdRgn.
```

Comment:
Please add a note here saying in future, `memberUuid` may get replaced by user provided name.
Otherwise, this statement doesn't make sense as `memberUuid` and `member` are one and the same.

Reply:
Currently I am putting memberUuid in both key and value. It will be change into a future for storing user provided member unique id. So in that case the value for member will also gets changed.

Reply:
Please put it as a comment in the code.